### PR TITLE
MMT-1008: Adds the ability to create variable collection associations

### PR DESCRIPTION
**Pull request by abbottry**

### CODE REVIEW

File: lib/tasks/drafts.rake [hunk 1]
```diff
@@ -21,6 +21,24 @@ namespace :drafts do
     end
   end
 
+  desc 'Load full draft into database'
+  task load_full_variable: :environment do
```
Comment (macrouch) [inline lib/tasks/drafts.rake]: `load_full_variable` is already in this file

Comment (abbottry) [reply lib/tasks/drafts.rake]: Hm. Look at that.

Comment (abbottry) [reply lib/tasks/drafts.rake]: Fixed that up -- good catch (the one i added was broke anyway 🤦‍♂️ )

File: app/controllers/collection_associations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  end
+
+  def new
+    add_breadcrumb 'New', new_variable_collection_association_path(params[:variable_id])
+
+    super
+
+    association_response = cmr_client.get_variables({ concept_id: params[:variable_id] }, token)
+
+    @previously_associated_collections = association_response.body.fetch('items', []).first.fetch('associations', {}).fetch('collections', []).map { |collection| collection['concept-id'] }
```
Comment (charleshuang80) [inline app/controllers/collection_associations_controller.rb]: You're parsing the association concept ids multiple times - is it worth making it into a method?

**Pull request merged**